data/multiplayer/scenarios/Wesbench_Shroud_Walk.cfg

```diff
@@ -2,7 +2,7 @@
 
 #ifdef DEBUG_MODE
 [binary_path]
-    path=data/campaigns/Heir_To_The_Throne
+    path=campaigns/Heir_To_The_Throne
```

Comment:
Removing the `data/` prefix is a very suspicious change. I'm not fundamentally against it, but it just seems like something that would break every add-on's binary path... unless of course it only affects mainline paths? If it does affect add-on paths, is there compatibility code to ensure add-ons still using it will continue to function?

Reply:
I don't know for sure wheterh  added it but there should definitley a compatibility path